From 33f0cd26507edc00488b3cdee9acb7a296b96edd Mon Sep 17 00:00:00 2001
From: Miroslav Benes <mbenes@suse.cz>
Date: Wed, 15 Mar 2017 15:12:34 +0100
Subject: [PATCH] kgr: remove unneeded kgr_needs_lazy_migration() s390x
 definition
Patch-mainline: not yet, kgraft
References: fate#313296

Function kgr_needs_lazy_migration() is not used anymore (the only caller
was removed with userspace stack detection hack). We omitted to remove its
s390x definition.

Signed-off-by: Miroslav Benes <mbenes@suse.cz>
---
 arch/s390/include/asm/kgraft.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/s390/include/asm/kgraft.h b/arch/s390/include/asm/kgraft.h
index 2bbe10220c23..d3ec8ed92c31 100644
--- a/arch/s390/include/asm/kgraft.h
+++ b/arch/s390/include/asm/kgraft.h
@@ -25,9 +25,4 @@ static inline void kgr_set_regs_ip(struct pt_regs *regs, unsigned long ip)
 	regs->psw.addr = ip;
 }
 
-static inline bool kgr_needs_lazy_migration(struct task_struct *p)
-{
-	return true;
-}
-
 #endif
-- 
2.12.0

